gammapy/spectrum/tests/test_powerlaw.py

```diff
@@ -26,6 +28,28 @@ def test_one():
     assert_allclose(I, 1)
 
 
+@requires_dependency('naima')
+def test_powerlaw_energy_flux():
+    """
+    Test energy flux computation for power law against numerical solution.
+    """
+    from naima.utils import trapz_loglog
+    e1 = Quantity(1, 'TeV')
+    e2 = Quantity(10, 'TeV')
+    einf = Quantity(1E10, 'TeV')
+    e = Quantity(1, 'TeV')
+    g = 2.3
+    I  = Quantity(1E-12, 'cm-2 s-1')
+    
+    eflux = power_law_energy_flux(I=I, g=g, e=e, e1=e1, e2=e2)
+
+    norm = power_law_flux(I=I, g=g, e=e, e1=e1, e2=einf)
+    f = lambda x: x * power_law_evaluate(x, norm, g, e)
+    val = integrate_loglog(f, e1, e2)
+
+    assert_quantity_allclose(val, eflux)
```

Comment:
Please add an assert on the computed result (i.e. the number).
If there's a bug or someone introduces one in the future in `integrate_loglog`, we won't notice with this test.

I would prefer to remove the little re-implementation of energy flux computation in this test function,
but if you prefer, OK to keep in addition to the assert on the number.